honeybee_energy/simulation/sizing.py

```diff
@@ -0,0 +1,135 @@
+# coding=utf-8
+"""Global sizing parameters dictating scaling factors for all zone peak loads."""
+from __future__ import division
+
+from ..reader import parse_idf_string
+from ..writer import generate_idf_string
+
+from honeybee.typing import float_positive
+
+
+class SizingParameter(object):
+    """Global sizing parameters dictating scaling factors for all zone peak loads.
+
+    Properties:
+        * heating_factor
+        * cooling_factor
+    """
+    __slots__ = ('_heating_factor', '_cooling_factor')
+
+    def __init__(self, heating_factor=1.25, cooling_factor=1.15):
+        """Initialize SizingParameter.
+
+        Args:
+            heating_factor: A number that will get multiplied by the peak heating load
+                for each zone in the model in order to size the heating system for
+                the model. Must be greater than 0. Default: 1.25.
+            cooling_factor: A number that will get multiplied by the peak cooling load
+                for each zone in the model in order to size the cooling system for
+                the model. Must be greater than 0. Default: 1.15.
+        """
+        self.heating_factor = heating_factor
+        self.cooling_factor = cooling_factor
+
+    @property
+    def heating_factor(self):
+        """Get or set a number that will get multiplied by the peak heating loads."""
+        return self._heating_factor
+
+    @heating_factor.setter
+    def heating_factor(self, value):
+        self._heating_factor = float_positive(value, 'sizing parameter heating factor')
```

Comment:
This will fail for value being 0.

Reply:
Hm. This type of situation may eventually require its own `typing` method since `float_in_range` won't really work here when 0.000000001 is still a valid input. For now, I'll just add an extra check to make sure it's not 0 but let's consider implementing a new `typing` method if more cases like this crop up in the future.